Makefile

```diff
@@ -29,17 +29,20 @@ PATCH=1
 OS := $(shell uname)
 ifeq ($(OS),Darwin) # MacOS X
   SHLIB_EXT = dylib
-  SHLIB_VERS_EXT = $(MAJOR).dylib
+  SHLIB_VERS_EXT = .$(MAJOR).$(SHLIB_EXT)
+else ifneq (,$(findstring MSYS_NT,$(OS))) # MinGW
```

Comment:
I don't think this is sufficient. Probably needs to copy the OS normalization from Julia

Reply:
This: https://github.com/JuliaLang/julia/blob/2f8cdd045cc6b7e77db14393e39cfee8ef4d4a66/Make.inc#L357-L368?